src/reaper_osara.cpp

```diff
@@ -4810,6 +4810,56 @@ void cmdInsertMarker(Command* command) {
 	outputMessage(format(translate("marker {} inserted"), number));
 }
 
+void cmdInsertProjectMarker(Command* command) {
+	cmdhInsertProjectMarker(command->gaccel.accel.cmd);
+}
+
+void cmdhInsertTakeMarker(int command) {
+	int preMarkers = 0;
+	for(int i = 0; i < CountSelectedMediaItems(nullptr); ++ i) {
+		MediaItem* item = GetSelectedMediaItem(nullptr, i);
+		MediaItem_Take* take = GetActiveTake(item);
+		preMarkers += GetNumTakeMarkers(take);
```

Comment:
Sorry, I missed this in the prior review. It's possible for an item to be empty and contain no takes, so you need to only do this if take is valid. Otherwise, we will possibly crash.

```suggestion
		if (take) {
			preMarkers += GetNumTakeMarkers(take);
		}
```